guestbook/php-mongodb/guestbook.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+error_reporting(E_ALL);
```

Comment:
Is this file content ported from `guestbook/php-redis/guestbook.php`?

Reply:
It [does appear so](https://github.com/kubernetes/examples/blob/master/guestbook/php-redis/guestbook.php), but I'm not sure. Why do you ask?